docs/source/essentials/server.md

```diff
@@ -25,99 +17,43 @@ To install, run:
 
     npm install --save apollo-server graphql
 
-## Importing
+<h2 id="creating">Creating a server</h2>
 
-Depending on whether we are creating a new application or an existing application, the method of importing `apollo-server` will vary slightly since Apollo Server must adapt to the semantics of existing servers (e.g. Express, Hapi, etc.)
+The fastest way to get started with GraphQL is by creating a new server. Apollo Server will set an Express server up for you as long as you provide it with `typeDefs`, which is a string representing your GraphQL schema, and `resolvers`, which is a map of functions that implement your schema.
```

Comment:
Apollo Server requires `typeDefs` and `resolvers` no matter what – that's not the criteria for whether or not it starts up an Express server.  The Express server is started when `listen` is called if you haven't attached `apollo-server` as middleware.

Reply:
Lots of `you` here.